src/taco-utils/tacoError.ts

```diff
@@ -76,7 +76,7 @@ module TacoUtility {
                     message = resources.getString(errorToken, args);
                 }
             } else {
```

Comment:
It looks like I can just remove this method entirely and use the "..." notation to invoke `resources.getString()` directly